.github/workflows/tests.yml

```diff
@@ -0,0 +1,60 @@
+name: tests
+
+on:
+  push: null
+  pull_request: null
```

Comment:
I'm not sure what null means here. I guess that means they'll always run in all PRs/branches, etc?